src/tests/ftest/cart/rpc/cart_rpc_one_node-test_crt_ep_abort.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2018-2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+'''
+
+from __future__ import print_function
+
+import sys
+
+from avocado import Test
+
+sys.path.append('./util')
+
+# Can't all this import before setting sys.path
```

Comment:
not sure i grok

Reply:
What it means is we have to override pylint's import ordering restriction here, because `cart_utils` can't be imported without `./utils` in the path.

There's also a typo: _Can't *call* this import before setting sys.path`_.